frontend/src/components/filter/activityReportFilters.js

```diff
@@ -347,6 +348,24 @@ export const ttaTypeFilter = {
   ),
 };
 
+export const cdiGrantsFilter = {
+  id: 'cdiGrants',
+  display: 'CDI grants',
+  conditions: FILTER_CONDITIONS,
+  defaultValues: {
+    is: 'active',
+    'is not': 'active',
+  },
```

Comment:
AC asks for the following:

Grant status
-     is / is not
-     Active / Inactive / Interim management (CDI)

Are we not doing that?

Reply:
Aren't these just the default values we show?

Reply:
"Grant status" not "CDI Grants" to be clear